test cases/common/160 reconfigure/meson.build

```diff
@@ -0,0 +1,5 @@
+project('reconfigure test', ['c'])
```

Comment:
This should probably be in `unit` tests instead of `common` tests because this will get run standalone otherwise and fail.